### PR TITLE
[NFR] add dataType JSON support for  MySQL dialect.

### PR DESCRIPTION
**Pull request by zGaron**
Hello!

* Type: new feature
* Link to issue: #12546, #12985, #13214

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I wrote some tests for this PR.

Small description of change:

Thanks

### CODE REVIEW

File: CHANGELOG-3.3.md [hunk 1]
```diff
@@ -21,3 +21,4 @@
 - Fixed `Phalcon\Mvc\Model::allowEmptyStringValues` to correct works with saving empty string values when DEFAULT not set in SQL
 - Fixed `Phalcon\Mvc\Model\Behavior\SoftDelete` to correctly update snapshots after deleting item
 - Fixed `Phalcon\Mvc\Model` to set old snapshot when no fields are changed when dynamic update is enabled
+- Added support JSON dataType for MySQL dialect. [#13216](https://github.com/phalcon/cphalcon/pull/13216)
```
Comment (Jurigag) [inline CHANGELOG-3.3.md]: Add it above in all `Added` entries, also we add url only pointing to issue, not pull.

**Comment by sergeyklay** [comment]
@zGaron Coul you please cover your changes by test. Also update please the change log. Thank you.

**Comment by zGaron** [comment]
@Jurigag how to let ci rerun, for new commits?

**Comment by sergeyklay** [comment]
@zGaron Could you please squash commits into one to keep history clean? Don't use `[ci skip]` for this single commit.

**Comment by zGaron** [comment]
create new PR #13225 ,  Thank you for guidance. @sergeyklay

**Pull request closed without merging**